grr/server/grr_response_server/gui/ui/components/flow_details/plugins/osquery_details.ts

```diff
@@ -15,58 +15,59 @@ import { isNonNull } from '@app/lib/preconditions';
   styleUrls: ['./osquery_details.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class OsqueryDetails extends Plugin implements OnDestroy {
-  readonly unsubscribe$ = new Subject<void>();
-
+export class OsqueryDetails extends Plugin {
   readonly flowRunning$ = this.flagByState(FlowState.RUNNING);
   readonly flowCompleted$ = this.flagByState(FlowState.FINISHED);
   readonly flowError$ = this.flagByState(FlowState.ERROR);
 
   readonly args$: Observable<OsqueryArgs> = this.flowListEntry$.pipe(
-      map((flowListEntry) => flowListEntry.flow.args as OsqueryArgs)
+      map(flowListEntry => flowListEntry.flow.args as OsqueryArgs),
   );
 
-  readonly osqueryResult$: Observable<OsqueryResult> = this.flowListEntry$.pipe(
-    flatMap((listEntry) => listEntry.resultSets),
-    flatMap((singleResultSet) => singleResultSet?.items),
+  readonly osqueryResults$: Observable<OsqueryResult> = this.flowListEntry$.pipe(
+    flatMap(listEntry => listEntry.resultSets),
+    flatMap(singleResultSet => singleResultSet?.items),
     filter(isNonNull),
-    map((singleItem) => singleItem.payload as OsqueryResult),
+    map(singleItem => singleItem.payload as OsqueryResult),
   );
 
-  readonly resultTable$ = this.osqueryResult$.pipe(
-    map((result) => result?.table),
+  readonly resultsTable$ = this.osqueryResults$.pipe(
+    map(result => result?.table),
```

Comment:
Please properly handle nullability: `osqueryResults$` emits non-nullable values only, but is accessed with `?` here and below.